layouts/redirect/list.html

```diff
@@ -0,0 +1,14 @@
+{{ $redirect_to := .Params.redirect_to }}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="refresh" content="0; url={{ $redirect_to }}">
+    <link rel="canonical" href="{{ $redirect_to }}">
+    <script>
+      window.location.href = "{{ $redirect_to }}";
```

Comment:
A minor nit, if we're doing time setup, might as well do a timeout in the head to the same amount of seconds as the refresh meta tag.